.github/ISSUE_TEMPLATE/02-question.yaml

```diff
@@ -0,0 +1,26 @@
+name: Question
```

Comment:
Shouldn't we rather encourage that questions all go to discussions? Those are much more visible there, especially after they have been answered.

Reply:
You're perfectly right, no need to duplicate.